### DISCUSSION TITLE
"Focusing" Status?

### DISCUSSION THREAD

**Discussion by StuartJGrant** [isaacrlevin/presencelight#608] [Ideas]
Might be a bit of tricky one, but it would be great if the app would recognise the "Focusing" status and allow you to set a different colour for this.  

Reason it may be tricky is that I think this is effectively DND and not a distinctive "status" value...not sure but thought i would throw the idea out there :)

Thanks for this tool, keep up the great work 👍

**Comment by isaacrlevin** [comment]
There is no way for me to track from the presence API Focusing. Here are the options that are available.

https://docs.microsoft.com/en-us/graph/api/resources/presence?view=graph-rest-1.0

**Comment by isaacrlevin** [comment]
Closing
